lib/parser.rb

```diff
@@ -0,0 +1,26 @@
+class Parser
+  COMMANDS = [:PLACE, :MOVE, :REPORT, :LEFT, :RIGHT]
+
+  def initialize(board)
+    @board = board
+  end
+
+  def parse(cmd)
+    cmd_arr = cmd.split(' ')
+    command = cmd_arr[0]
+    args = cmd_arr[1]
+
+    validate_command(command)
+    args ? [command.downcase] + args.split(',') : [command.downcase]
```

Comment:
Hmm, I actually find the ternary operator very readable in almost any form :)) I'll consider to refactor.